src/settings.ts

```diff
@@ -43,5 +43,8 @@ export const settings = {
   "editor.formatOnSave": false,
 
   "//k": "Disable preview so we can open all files at once",
-  "workbench.editor.enablePreview": false
+  "workbench.editor.enablePreview": false,
+
+  "//l": "Use the Markdown Preview pane to view Markdown files",
+  "workbench.useMdPreview": false
```

Comment:
I don't think this is necessary. These are the default settings. The new configuration doesn't have to be here 😉